AzureDay.WebApp/AzureDay.WebApp.Database/Services/CountryService.cs

```diff
@@ -0,0 +1,19 @@
+﻿using System.Linq;
+using System.Collections.Generic;
+using AzureDay.WebApp.Database.Entities;
+
+namespace AzureDay.WebApp.Database.Services
+{
+    public sealed class CountryService : BaseService<CountryEntity, int>
+    {
+        protected override List<CountryEntity> PopulateStorage()
+        {
+            return new List<CountryEntity>
+            {
+                new CountryEntity {Id = 1, ImageUrl = "https://azureday2017ua.blob.core.windows.net/images/flags/ua.png", Title = Localization.App.Service.Country.Ukraine}
```

Comment:
please add this image to source code repo, we should not use external resources except database (table storage)